scapy/layers/tls/record.py

```diff
@@ -180,6 +181,7 @@ def i2m(self, pkt, p):
                     pkt.type = 22
                 elif isinstance(p, TLSApplicationData):
                     pkt.type = 23
+
```

Comment:
Please do not introduce useless changes.

Reply:
I will also fix this and the others in my next commit.

Reply:
Thanks a lot!